cli/dev/utils.go

```diff
@@ -699,6 +704,20 @@ func stopContainers(dc *common.DeploymentConfig, cw *container.ContainerWorker,
 			cw.ResourcesRemove([]string{msId})
 		}
 	}
+
+	// Remove the UDS socket file
```

Comment:
Unfortunately, this call happens before all of a service's dependencies have been stopped, so the directory cleanup really needs to be done: https://github.com/open-horizon/anax/blob/master/cli/dev/utils.go#L633 or later.

Reply:
Moved cleaning stuff to specified location. Thanks